tools/kmod/sof_remove.sh

```diff
@@ -14,7 +14,7 @@ remove_module() {
     fi
 }
 
-remove_module snd_usb_audio
+remove_module -f snd_usb_audio
```

Comment:
Why do we test snd_usb_audio kernel module remove? I think USB audio driver kernel module remove is out of scope.
BTW: have you checked pulseaudio?